mico-core/src/main/java/io/github/ust/mico/core/resource/OpenFaasResource.java

```diff
@@ -63,4 +74,22 @@
             throw new ResponseStatusException(HttpStatus.GATEWAY_TIMEOUT, e.getMessage());
         }
     }
+
+    @GetMapping(EXTERNAL_ADDRESS_PATH)
```

Comment:
I would expect the general information about the openfaas instance when i request the `OPEN_FAAS_BASE_PATH`. Just the external url does not really need an extra subpath.
(This should only be changed if it is really easy (<10min) to change it)